azext_iot/central/params.py

```diff
@@ -120,12 +129,48 @@ def load_central_arguments(self, _):
             help="Tenant ID for service principal to be added to the app. Object ID must also be specified. ",
         )
 
+    with self.argument_context("iot central app api-token") as context:
         context.argument(
             "token_id",
             options_list=["--token-id", "--tkid"],
             help="Unique ID for the API token. ",
         )
 
+    with self.argument_context("iot central app cde") as context:
+        context.argument("sources", arg_type=sources)
+        context.argument(
+            "display_name",
+            options_list=["--display-name", "--dn"],
+            help="Display name of the continuous data export.",
+        )
+        context.argument(
+            "enable",
+            options_list=["--enable", "-e"],
+            arg_type=get_three_state_flag(),
+            help="Boolean indicating whether the continuous data export should be running or not. ",
+        )
+        context.argument(
+            "ep_conn",
+            options_list=["--ep-conn", "-c"],
+            help="Connection string for the endpoint. ",
+        )
+        context.argument(
+            "entity_name",
+            options_list=["--entity-name", "--en"],
+            help="Name of entity pointing at Eg: container_name, queue_name, etc.. ",
```

Comment:
I think this description for --entity-name needs to be more clear and related the grammar seems off.